app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Login</h1>
+   <%= form_tag '/login' do %>
```

Comment:
@idprietop we should not use routes in this way, there are route helper methods to do this. Please correct

Reply:
Ok, I already made the change.